MatchThreeBoardTest.java

```diff
@@ -120,8 +121,74 @@ public void nullTokens2Test()
         exception.expect(IllegalArgumentException.class);
         new MatchThreeBoard(null, "  ;  ");
     }
+    
+    @Test
+    public void nullPositionTest1()
+    {
+        exception.expect(IllegalArgumentException.class);
```

Comment:
The `exception.expect`-clause should come directly before the line that is expected to throw.